src/components/ItemsList.js

```diff
@@ -2,24 +2,36 @@ import React from 'react';
 import firebase from '../lib/firebase';
 import Nav from './Nav';
 import { useCollectionData } from 'react-firebase-hooks/firestore';
+import { useHistory } from 'react-router-dom';
+import '../styles/ItemsList.css';
 
 const db = firebase.firestore().collection('shopping_list');
 
 const ItemsList = () => {
   const userToken = localStorage.getItem('token');
+  const history = useHistory();
 
   const [shoppingList, loading, error] = useCollectionData(
     db.where('token', '==', userToken),
   );
 
+  const handleRedirect = () => {
+    history.push('/additem');
+  };
+
   return (
     <div>
       <h1>Your Shopping List</h1>
       <div>
         {loading && <p>Loading...</p>}
         {error && <p>An error has occured...</p>}
         {shoppingList && !shoppingList.length && (
-          <p>You haven't created a shopping list yet...</p>
+          <div className="add-item">
+            <p>You haven't created a shopping list yet...</p>
+            <button type="submit" onClick={handleRedirect}>
+              Add Item
```

Comment:
The button name is totally fine here but since we are using `Add Item` as one of our navbars maybe it would be better to rename it as something like: 
`Create shopping list` or `Add first item`. These are just examples. The idea here is that more clearer names will help users to better navigate the app. This is just a suggestion :)